js/background.js

```diff
@@ -120,31 +127,69 @@ chrome.contextMenus.create({
 // Add ATB param
 chrome.webRequest.onBeforeRequest.addListener(
     function (e) {
-      // Only change the URL if there is no ATB param specified.
-      if (e.url.indexOf('atb=') !== -1) {
-        return;
-      }
 
-      // Only change the URL if there is an ATB saved in localStorage
-      if (localStorage['atb'] === undefined) {
-        return;
+      // Add ATB for DDG URLs, otherwise block trackers
+      if (e.url.search('/duckduckgo\.com') !== -1) {
+          // Only change the URL if there is no ATB param specified.
+          if (e.url.indexOf('atb=') !== -1) {
+            return;
+          }
+
+          // Only change the URL if there is an ATB saved in localStorage
+          if (localStorage['atb'] === undefined) {
+            return;
+          }
+        
+          var newURL = e.url + "&atb=" + localStorage['atb'];
+          return {
+            redirectUrl: newURL
+          };
+      } 
+      else {
+          chrome.tabs.query({
```

Comment:
It's async and I'm using localStorage to read it later outside of chrome.tabs.query